### PR TITLE
feat(plasma-tokens): Add `src` directory to commit and remove that from .gitignore

### PR DESCRIPTION
**Pull request by neretin-trike**
Исправил метод ` generateFile`, который в качестве аргумент по умолчанию принимал `undefined` и добавлял это значение в конец сгенерированных файлов.

Обновил название темы `sbermarket_winelab` на `sbermarket_wlbusiness`.

Добавил диреткорию `src` в репозиторий для того, чтобы появилась возможность отслеживать изменения автоматической генерации тем и токенов.  Так же необходимо было добавить авто линт после шага со сборкой для применения форматирования в созданных файлах.

:baby_chick: Download canary assets:

[colors--canary.488.4757355253.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.488.4757355253.xml)  
[color_sberHealth_ios-swift--canary.488.4757355253.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.488.4757355253.swift)  
[color_sberHealth_kotlin--canary.488.4757355253.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.488.4757355253.kt)  
[color_sberHealth_react-native--canary.488.4757355253.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.488.4757355253.ts)  
[color_sbermarket_business_ios-swift--canary.488.4757355253.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.488.4757355253.swift)  
[color_sbermarket_business_kotlin--canary.488.4757355253.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.488.4757355253.kt)  
[color_sbermarket_business_react-native--canary.488.4757355253.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.488.4757355253.ts)  
[color_sbermarket_ios-swift--canary.488.4757355253.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.488.4757355253.swift)  
[color_sbermarket_kotlin--canary.488.4757355253.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.488.4757355253.kt)  
[color_sbermarket_metro_ios-swift--canary.488.4757355253.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.488.4757355253.swift)  
[color_sbermarket_metro_kotlin--canary.488.4757355253.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.488.4757355253.kt)  
[color_sbermarket_metro_react-native--canary.488.4757355253.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.488.4757355253.ts)  
[color_sbermarket_react-native--canary.488.4757355253.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.488.4757355253.ts)  
[color_sbermarket_selgros_ios-swift--canary.488.4757355253.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.488.4757355253.swift)  
[color_sbermarket_selgros_kotlin--canary.488.4757355253.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.488.4757355253.kt)  
[color_sbermarket_selgros_react-native--canary.488.4757355253.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.488.4757355253.ts)  
[color_sbermarket_wlbusiness_ios-swift--canary.488.4757355253.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.488.4757355253.swift)  
[color_sbermarket_wlbusiness_kotlin--canary.488.4757355253.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.488.4757355253.kt)  
[color_sbermarket_wlbusiness_react-native--canary.488.4757355253.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.488.4757355253.ts)  
[color_sberprime_ios-swift--canary.488.4757355253.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.488.4757355253.swift)  
[color_sberprime_kotlin--canary.488.4757355253.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.488.4757355253.kt)  
[color_sberprime_react-native--canary.488.4757355253.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.488.4757355253.ts)  
[PlasmaTokensColor--canary.488.4757355253.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.488.4757355253.swift)  
[shadow_sbermarket_react-native--canary.488.4757355253.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.488.4757355253.ts)  
[typo_mage_ios-swift--canary.488.4757355253.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.488.4757355253.swift)  
[typo_mage_kotlin--canary.488.4757355253.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.488.4757355253.kt)  
[typo_mage_react-native--canary.488.4757355253.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.488.4757355253.ts)  
[typo_plasma_ios-swift--canary.488.4757355253.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.488.4757355253.swift)  
[typo_plasma_kotlin--canary.488.4757355253.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.488.4757355253.kt)  
[typo_plasma_react-native--canary.488.4757355253.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.488.4757355253.ts)  
[typo_ruler_ios-swift--canary.488.4757355253.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.488.4757355253.swift)  
[typo_ruler_kotlin--canary.488.4757355253.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.488.4757355253.kt)  
[typo_ruler_react-native--canary.488.4757355253.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.488.4757355253.ts)  
[typo_sage_ios-swift--canary.488.4757355253.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.488.4757355253.swift)  
[typo_sage_kotlin--canary.488.4757355253.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.488.4757355253.kt)  
[typo_sage_react-native--canary.488.4757355253.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.488.4757355253.ts)  
[typo_sbermarket_ios-swift--canary.488.4757355253.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.488.4757355253.swift)  
[typo_sbermarket_kotlin--canary.488.4757355253.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.488.4757355253.kt)  
[typo_sbermarket_react-native--canary.488.4757355253.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.488.4757355253.ts)  
[typo_soulmate_ios-swift--canary.488.4757355253.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.488.4757355253.swift)  
[typo_soulmate_kotlin--canary.488.4757355253.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.488.4757355253.kt)  
[typo_soulmate_react-native--canary.488.4757355253.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.488.4757355253.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.203.0-canary.488.4757355253.0
  npm install @salutejs/plasma-core@1.110.0-canary.488.4757355253.0
  npm install @salutejs/plasma-hope@1.203.0-canary.488.4757355253.0
  npm install @salutejs/plasma-icons@1.139.0-canary.488.4757355253.0
  npm install @salutejs/plasma-temple@1.153.0-canary.488.4757355253.0
  npm install @salutejs/plasma-tokens-b2b@1.16.0-canary.488.4757355253.0
  npm install @salutejs/plasma-tokens-b2c@0.25.0-canary.488.4757355253.0
  npm install @salutejs/plasma-tokens-web@1.31.0-canary.488.4757355253.0
  npm install @salutejs/plasma-tokens@1.46.0-canary.488.4757355253.0
  npm install @salutejs/plasma-ui@1.185.0-canary.488.4757355253.0
  npm install @salutejs/plasma-web@1.203.0-canary.488.4757355253.0
  npm install @salutejs/plasma-cy-utils@0.53.0-canary.488.4757355253.0
  npm install @salutejs/plasma-sb-utils@0.108.0-canary.488.4757355253.0
  npm install @salutejs/plasma-tokens-android@2.53.0-canary.488.4757355253.0
  npm install @salutejs/plasma-tokens-ios-swift@2.53.0-canary.488.4757355253.0
  npm install @salutejs/plasma-tokens-utils@0.22.0-canary.488.4757355253.0
  # or 
  yarn add @salutejs/plasma-b2c@1.203.0-canary.488.4757355253.0
  yarn add @salutejs/plasma-core@1.110.0-canary.488.4757355253.0
  yarn add @salutejs/plasma-hope@1.203.0-canary.488.4757355253.0
  yarn add @salutejs/plasma-icons@1.139.0-canary.488.4757355253.0
  yarn add @salutejs/plasma-temple@1.153.0-canary.488.4757355253.0
  yarn add @salutejs/plasma-tokens-b2b@1.16.0-canary.488.4757355253.0
  yarn add @salutejs/plasma-tokens-b2c@0.25.0-canary.488.4757355253.0
  yarn add @salutejs/plasma-tokens-web@1.31.0-canary.488.4757355253.0
  yarn add @salutejs/plasma-tokens@1.46.0-canary.488.4757355253.0
  yarn add @salutejs/plasma-ui@1.185.0-canary.488.4757355253.0
  yarn add @salutejs/plasma-web@1.203.0-canary.488.4757355253.0
  yarn add @salutejs/plasma-cy-utils@0.53.0-canary.488.4757355253.0
  yarn add @salutejs/plasma-sb-utils@0.108.0-canary.488.4757355253.0
  yarn add @salutejs/plasma-tokens-android@2.53.0-canary.488.4757355253.0
  yarn add @salutejs/plasma-tokens-ios-swift@2.53.0-canary.488.4757355253.0
  yarn add @salutejs/plasma-tokens-utils@0.22.0-canary.488.4757355253.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-488/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-488/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-488/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-488/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-488/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-488/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-488/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-488/temple-storybook/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-488/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-488/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-488/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-488/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-488/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Review by Yakutoc — APPROVED** [review]

**Review by kayman233 — APPROVED** [review]

File: utils/plasma-tokens-utils/src/generators/generateFile.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { ROBO_COMMENT } from '../constants';
 /**
  * Создает дескриптор на файл с определенным именем и контентом.
  */
-export const generateFile = (name: string, content: string | object, deprecated?: string) => {
+export const generateFile = (name: string, content: string | object, deprecated = '') => {
```
Comment (Yeti-or) [inline utils/plasma-tokens-utils/src/generators/generateFile.ts:6]: 👍

File: packages/plasma-tokens/data/themes/sbermarket_wlbusiness.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "config": {
-        "name": "sbermarket_winelab",
+        "name": "sbermarket_wlbusiness",
```
Comment (Yeti-or) [inline packages/plasma-tokens/data/themes/sbermarket_wlbusiness.json:3]: 😫

**Review by Yeti-or — APPROVED** [review]

**Pull request merged**